easybuild/toolchains/foss.py

```diff
@@ -40,7 +41,7 @@
 class Foss(Gompi, OpenBLAS, FlexiBLAS, ScaLAPACK, Fftw):
     """Compiler toolchain with GCC, OpenMPI, OpenBLAS, ScaLAPACK and FFTW."""
     NAME = 'foss'
-    SUBTOOLCHAIN = [Gompi.NAME, Golf.NAME]
+    SUBTOOLCHAIN = [Gompi.NAME, Golf.NAME, Gfbf.NAME]
```

Comment:
Need tio add Golf and Gfbf calls to the banned_linked_shaed_libs too

Reply:
I rolled back the changes done in commit 26e3e22 in commit 029695ea4, since it's not needed at all: the listed of banned linked shared libs is extended via the individual toolchain components (`OpenBLAS`/`FlexiBLAS`, ...) already!